src/elements/hx-file-control/index.spec.js

```diff
@@ -0,0 +1,34 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-file-control> component tests
+ *
+ * @type HXFileControlElement
+ *
+ */
+describe('<hx-file-control> component tests', () => {
+    const template = '<hx-file-control>';
+
+    describe('test instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXFileControlElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXFileControlElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXFileControlElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
```

Comment:
Please add a test to verify that  `<hx-file-control>` does not have a Shadow DOM.

Reply:
Also please update `master`, and rebase before pushing up your changes.